datalad/utils.py

```diff
@@ -1983,28 +1995,31 @@ def try_multiple_dec(f, ntrials=None, duration=0.1, exceptions=None, increment_t
     if not ntrials:
         # Life goes fast on proper systems, no need to delay it much
         ntrials = 100 if on_windows else 10
-
+    if logger is None:
+        def logger(*args, **kwargs):
+            return lgr.log(5, *args, **kwargs)
     assert increment_type in {None, 'exponential'}
 
     @wraps(f)
-    def wrapped(*args, **kwargs):
+    def _wrap_try_multiple_dec(*args, **kwargs):
```

Comment:
note: while at it I have renamed this generic `wrapped` which generally should not be used for DX while observing the traceback etc